### DISCUSSION TITLE
How to show in OBS Studio

### DISCUSSION THREAD

**Discussion by Sabcoll** [pubpub-zz/ppInk#275] [Q&A]
Hi Everyone,

i hope somebody can come up with a How-To for dummies.
I have no clue about websockets and I don't want to control anything with obs or anything to control obs.

All i want to archieve is to show ppInk in my obs, no matter if I record or stream .. or do nothing at all and just enjoy the preview.
How do I add ppInk as a Source to OBS?

Please answer in steps for a stupid user like me, who doesn't know what a websocket is!
Thanks in advance, highly appreciated!

**Comment by pubpub-zz** [comment]
My first advices:
a) capture the whole screen and you will get the applications with the annotations over. Nothing special to do
else
b) capture ppink like any standard window
to be sure  to get the good name
start the inking mode (click on the icon)
go to pointer mode (click on the arrow button)
you can then configure obs.
this is my setup
![image](https://github.com/pubpub-zz/ppInk/assets/4083478/63bac794-fcbe-46a2-bf79-e947a3db0a98)
